Description: Update the list of unimplemented functions
Origin: backport, https://hg.savannah.gnu.org/hgweb/octave/rev/d41d137af059
Bug: https://savannah.gnu.org/bugs/index.php?55094
Bug-Debian: https://bugs.debian.org/914391
Last-Update: 2019-01-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -r e95e17d6b6dc -r d41d137af059 scripts/help/__unimplemented__.m
--- a/scripts/help/__unimplemented__.m	Fri Dec 21 14:17:34 2018 -0800
+++ b/scripts/help/__unimplemented__.m	Fri Dec 21 14:24:12 2018 -0800
@@ -367,97 +367,118 @@
       txt = check_package (fcn, "signal");
 
     ## statistics
-    case {"addedvarplot", "addlevels", "addTerms", "adtest", ...
-          "andrewsplot", "anova1", "anova2", "anovan", "ansaribradley", ...
-          "aoctool", "barttest", "bbdesign", "betafit", "betalike", ...
-          "betastat", "binofit", "binostat", "biplot", "bootci", "bootstrp", ...
-          "boxplot", "candexch", "candgen", "canoncorr", "capability", ...
-          "capaplot", "caseread", "casewrite", "ccdesign", "cdf", ...
-          "cdfplot", "cell2dataset", "chi2gof", "chi2stat", "cholcov", ...
+    case {"addedvarplot", "addlevels", "addTerms", "adtest", "andrewsplot", ...
+          "anova", "anova1", "anova2", "anovan", "ansaribradley", ...
+          "aoctool", "bartlett_test", "barttest", "bbdesign", "betacdf", ...
+          "betafit", "betainv", "betalike", "betapdf", "betarnd", ...
+          "betastat", "binocdf", "binofit", "binoinv", "binopdf", ...
+          "binornd", "binostat", "biplot", "bootci", "bootstrp", "boxplot", ...
+          "candexch", "candgen", "canoncorr", "capability", "capaplot", ...
+          "caseread", "casewrite", "cauchy_cdf", "cauchy_inv", ...
+          "cauchy_pdf", "cauchy_rnd", "ccdesign", "cdf", "cdfplot", ...
+          "cell2dataset", "chi2cdf", "chi2gof", "chi2inv", "chi2pdf", ...
+          "chi2rnd", "chi2stat", "chisquare_test_homogeneity", ...
+          "chisquare_test_independence", "cholcov", ...
           "ClassificationBaggedEnsemble", "ClassificationDiscriminant", ...
           "ClassificationEnsemble", "ClassificationKNN", ...
           "ClassificationPartitionedEnsemble", ...
           "ClassificationPartitionedModel", "ClassificationTree", ...
-          "classify", "classregtree", "cluster", "clusterdata", "cmdscale", ...
-          "coefCI", "coefTest",  ...
-          "combnk", "compact", ...
+          "classify", "classregtree", "cloglog", "cluster", "clusterdata", ...
+          "cmdscale", "coefCI", "coefTest", "combnk", "compact", ...
           "CompactClassificationDiscriminant", ...
           "CompactClassificationEnsemble", "CompactClassificationTree", ...
           "CompactRegressionEnsemble", "CompactRegressionTree", ...
           "CompactTreeBagger", "compare", "confusionmat", "controlchart", ...
           "controlrules", "cophenet", "copulacdf", "copulafit", ...
-          "copulaparam", "copulapdf", "copularnd", "copulastat", "cordexch", ...
-          "corrcov", "covarianceParameters", "coxphfit", "createns", ...
-          "crosstab", "crossval", "cvpartition", "datasample", "dataset", ...
-          "dataset2cell", "dataset2struct", "dataset2table", "datasetfun", ...
-          "daugment", "dcovary", "dendrogram", "designMatrix", ...
-          "devianceTest", "dfittool",  ...
+          "copulaparam", "copulapdf", "copularnd", "copulastat", ...
+          "cordexch", "corrcov", "cor_test", "covarianceParameters", ...
+          "coxphfit", "createns", "crosstab", "crossval", "cvpartition", ...
+          "datasample", "dataset", "dataset2cell", "dataset2struct", ...
+          "dataset2table", "datasetfun", "daugment", "dcovary", ...
+          "dendrogram", "designMatrix", "devianceTest", "dfittool", ...
           "disttool", "droplevels", "dummyvar", "dwtest", "ecdf", ...
           "ecdfhist", "evalclusters", "evcdf", "evfit", "evinv", "evlike", ...
-          "evpdf", "evrnd", "evstat", "ExhaustiveSearcher", "expfit", ...
-          "explike", "export", "expstat", "factoran", ...
-          "ff2n", "fitdist", "fitensemble", "fitglm", "fitlm", ...
-          "fitlme", "fitlmematrix", "fitnlm", "fitted", "fixedEffects", ...
-          "fracfact", "fracfactgen", "friedman", "fsurfht", "fullfact", ...
-          "gagerr", "gamfit", "gamlike", "gamstat", ...
-          "GeneralizedLinearModel", "geomean", "geostat", "getlabels", ...
-          "getlevels", "gevcdf", "gevfit", "gevinv", "gevlike", "gevpdf", ...
-          "gevrnd", "gevstat", "gline", "glmfit", "glmval", "glyphplot", ...
+          "evpdf", "evrnd", "evstat", "ExhaustiveSearcher", "expcdf", ...
+          "expfit", "expinv", "explike", "export", "exppdf", "exprnd", ...
+          "expstat", "factoran", "fcdf", "ff2n", "finv", "fitdist", ...
+          "fitensemble", "fitglm", "fitlm", "fitlme", "fitlmematrix", ...
+          "fitnlm", "fitted", "fixedEffects", "fpdf", "fracfact", ...
+          "fracfactgen", "friedman", "frnd", "fsurfht", "fullfact", ...
+          "f_test_regression", "gagerr", "gamcdf", "gamfit", "gaminv", ...
+          "gamlike", "gampdf", "gamrnd", "gamstat", ...
+          "GeneralizedLinearModel", "geocdf", "geoinv", "geomean", ...
+          "geopdf", "geornd", "geostat", "getlabels", "getlevels", ...
+          "gevcdf", "gevfit", "gevinv", "gevlike", "gevpdf", "gevrnd", ...
+          "gevstat", "gline", "glmfit", "glmval", "glyphplot", ...
           "gmdistribution", "gname", "gpcdf", "gpfit", "gpinv", "gplike", ...
           "gplotmatrix", "gppdf", "gprnd", "gpstat", "grp2idx", "grpstats", ...
           "gscatter", "haltonset", "harmmean", "hist3", "histfit", ...
           "hmmdecode", "hmmestimate", "hmmgenerate", "hmmtrain", ...
-          "hmmviterbi", "hougen", "hygestat", "icdf", ...
+          "hmmviterbi", "hotelling_test", "hotelling_test_2", "hougen", ...
+          "hygecdf", "hygeinv", "hygepdf", "hygernd", "hygestat", "icdf", ...
           "inconsistent", "interactionplot", "invpred", "islevel", ...
           "ismissing", "isundefined", "iwishrnd", "jackknife", "jbtest", ...
           "johnsrnd", "join", "KDTreeSearcher", "kmeans", "knnsearch", ...
-          "kruskalwallis", "ksdensity", "kstest", "kstest2", "labels", ...
-          "lasso", "lassoglm", "lassoPlot", "levelcounts", "leverage", ...
-          "lhsdesign", "lhsnorm", "lillietest", "LinearMixedModel", ...
-          "LinearModel", "linhyptest", "linkage", "lognfit", "lognlike", ...
+          "kolmogorov_smirnov_cdf", "kolmogorov_smirnov_test", ...
+          "kolmogorov_smirnov_test_2", "kruskalwallis", ...
+          "kruskal_wallis_test", "ksdensity", "kstest", "kstest2", ...
+          "labels", "laplace_cdf", "laplace_inv", "laplace_pdf", ...
+          "laplace_rnd", "lasso", "lassoglm", "lassoPlot", "levelcounts", ...
+          "leverage", "lhsdesign", "lhsnorm", "lillietest", ...
+          "LinearMixedModel", "LinearModel", "linhyptest", "linkage", ...
+          "logistic_cdf", "logistic_inv", "logistic_pdf", ...
+          "logistic_regression", "logistic_rnd", "logit", "logncdf", ...
+          "lognfit", "logninv", "lognlike", "lognpdf", "lognrnd", ...
           "lognstat", "lsline", "mad", "mahal", "maineffectsplot", ...
-          "makedist", "manova1", "manovacluster", "mat2dataset", "mdscale", ...
-          "mergelevels", "mhsample", "mle", "mlecov", "mnpdf", "mnrfit", ...
-          "mnrnd", "mnrval", "multcompare", "multivarichart", "mvncdf", ...
-          "mvnpdf", "mvnrnd", "mvregress", "mvregresslike", "mvtcdf", ...
-          "mvtpdf", "mvtrnd", "NaiveBayes", "nancov", "nanmax", "nanmean", ...
-          "nanmedian", "nanmin", "nanstd", "nansum", "nanvar", "nbinfit", ...
-          "nbinstat", "ncfcdf", "ncfinv", "ncfpdf", "ncfrnd", "ncfstat", ...
-          "nctcdf", "nctinv", "nctpdf", "nctrnd", "nctstat", "ncx2cdf", ...
-          "ncx2inv", "ncx2pdf", "ncx2rnd", "ncx2stat", "negloglik", ...
-          "nlinfit", "nlintool", "nlmefit", "nlmefitsa", ...
+          "makedist", "manova", "manova1", "manovacluster", "mat2dataset", ...
+          "mcnemar_test", "mdscale", "mergelevels", "mhsample", "mle", ...
+          "mlecov", "mnpdf", "mnrfit", "mnrnd", "mnrval", "multcompare", ...
+          "multivarichart", "mvncdf", "mvnpdf", "mvnrnd", "mvregress", ...
+          "mvregresslike", "mvtcdf", "mvtpdf", "mvtrnd", "NaiveBayes", ...
+          "nancov", "nanmax", "nanmean", "nanmedian", "nanmin", "nanstd", ...
+          "nansum", "nanvar", "nbincdf", "nbinfit", "nbininv", "nbinpdf", ...
+          "nbinrnd", "nbinstat", "ncfcdf", "ncfinv", "ncfpdf", "ncfrnd", ...
+          "ncfstat", "nctcdf", "nctinv", "nctpdf", "nctrnd", "nctstat", ...
+          "ncx2cdf", "ncx2inv", "ncx2pdf", "ncx2rnd", "ncx2stat", ...
+          "negloglik", "nlinfit", "nlintool", "nlmefit", "nlmefitsa", ...
           "nlparci", "nlpredci", "nnmf", "nominal", "NonLinearModel", ...
-          "normfit", "normlike", "normplot", "normspec", "normstat", ...
+          "normcdf", "normfit", "norminv", "normlike", "normpdf", ...
+          "normplot", "normrnd", "normspec", "normstat", ...
           "optimalleaforder", "ordinal", "parallelcoords", "paramci", ...
           "paretotails", "partialcorr", "partialcorri", "pca", "pcacov", ...
-          "pcares", "pdf", "pdist", "pdist2", "pearsrnd", ...
-          "perfcurve", "plotAdded", "plotAdjustedResponse", ...
-          "plotDiagnostics", ...
-          "plotEffects", "plotInteraction", "plotResiduals", ...
-          "plotSlice", ...
-          "plsregress", "poissfit", "poisstat", ...
-          "polyconf", "polytool", "ppca", "predict", ...
-          "princomp", ...
-          "ProbDistUnivKernel", "ProbDistUnivParam", "probplot", ...
-          "procrustes", "proflik", "qrandset", "qrandstream", "random", ...
-          "randomEffects", ...
-          "randsample", "randtool", "rangesearch", "ranksum", "raylcdf", ...
-          "raylfit", "raylinv", "raylpdf", "raylrnd", "raylstat", "rcoplot", ...
-          "refcurve", "refline", "regress", "RegressionBaggedEnsemble", ...
+          "pcares", "pdf", "pdist", "pdist2", "pearsrnd", "perfcurve", ...
+          "plotAdded", "plotAdjustedResponse", "plotDiagnostics", ...
+          "plotEffects", "plotInteraction", "plotResiduals", "plotSlice", ...
+          "plsregress", "poisscdf", "poissfit", "poissinv", "poisspdf", ...
+          "poissrnd", "poisstat", "polyconf", "polytool", "ppca", ...
+          "prctile", "predict", "princomp", "ProbDistUnivKernel", ...
+          "ProbDistUnivParam", "probit", "probplot", "procrustes", ...
+          "proflik", "prop_test_2", "qqplot", "qrandset", "qrandstream", ...
+          "random", "randomEffects", "randsample", "randtool", ...
+          "rangesearch", "ranksum", "raylcdf", "raylfit", "raylinv", ...
+          "raylpdf", "raylrnd", "raylstat", "rcoplot", "refcurve", ...
+          "refline", "regress", "RegressionBaggedEnsemble", ...
           "RegressionEnsemble", "RegressionPartitionedEnsemble", ...
           "RegressionPartitionedModel", "RegressionTree", "regstats", ...
-          "relieff", "removeTerms", "residuals", "response", ...
-          "ridge", "robustdemo", "robustfit", "rotatefactors", "rowexch", ...
-          "rsmdemo", "rstool", "runstest", "sampsizepwr", "scatterhist", ...
-          "sequentialfs", "setlabels", "signrank", "signtest", "silhouette", ...
-          "slicesample", "sobolset", "squareform", "statget", "statset", ...
-          "step", "stepwise", "stepwisefit", "stepwiseglm", ...
-          "stepwiselm", "struct2dataset", "surfht", "svmclassify", ...
-          "svmtrain", "table2dataset", "tabulate", "tblread", "tblwrite", ...
-          "tdfread", "tiedrank", "TreeBagger", "trimmean", "truncate", ...
-          "tstat", "ttest", "ttest2", "unidstat", "unifit", "unifstat", ...
-          "vartest", "vartest2", "vartestn", "wblfit", "wbllike", "wblplot", ...
-          "wblstat", "wishrnd", "x2fx", "xlsread", "xptread", "ztest"}
+          "relieff", "removeTerms", "residuals", "response", "ridge", ...
+          "robustdemo", "robustfit", "rotatefactors", "rowexch", "rsmdemo", ...
+          "rstool", "runstest", "run_test", "sampsizepwr", "scatterhist", ...
+          "sequentialfs", "setlabels", "signrank", "signtest", "sign_test", ...
+          "silhouette", "slicesample", "sobolset", "squareform", "statget", ...
+          "statset", "stdnormal_cdf", "stdnormal_inv", "stdnormal_pdf", ...
+          "stdnormal_rnd", "step", "stepwise", "stepwisefit", ...
+          "stepwiseglm", "stepwiselm", "struct2dataset", "surfht", ...
+          "svmclassify", "svmtrain", "table2dataset", "tabulate", ...
+          "tblread", "tblwrite", "tcdf", "tdfread", "tiedrank", "tinv", ...
+          "tpdf", "TreeBagger", "trimmean", "trnd", "truncate", "tstat", ...
+          "ttest", "ttest2", "t_test", "t_test_2", "t_test_regression", ...
+          "unidcdf", "unidinv", "unidpdf", "unidrnd", "unidstat", ...
+          "unifcdf", "unifinv", "unifit", "unifpdf", "unifrnd", "unifstat", ...
+          "u_test", "vartest", "vartest2", "vartestn", "var_test", ...
+          "wblcdf", "wblfit", "wblinv", "wbllike", "wblpdf", "wblplot", ...
+          "wblrnd", "wblstat", "welch_test", "wienrnd", "wilcoxon_test", ...
+          "wishrnd", "x2fx", "xlsread", "xptread", "ztest", "z_test", ...
+          "z_test_2"}
       txt = check_package (fcn, "statistics");
 
     ## symbolic
